guard/src/rules/values_tests.rs

```diff
@@ -177,7 +177,6 @@ fn test_query_on_value() -> Result<()> {
         if let PathAwareValue::Map(_index) = each {
             continue;
         }
-        assert!(false);
```

Comment:
I think this might be an exit for a negative test? Like if code reaches here we want to fail the test right?

Reply:
Possibly, but then wouldnt it be best to do something like

```
assert!(matches!(PathAwareValue::Map(_), each)); 
```